src/python/pants/notes/1.11.x.rst

```diff
@@ -0,0 +1,249 @@
+API Changes
```

Comment:
This may need to have a header then note 1.11.0rc0 similar to https://github.com/pantsbuild/pants/commit/a9c21a5c1485e4069dfb8bd0a298b3ee745ccfd8#diff-b00d1a01cbdc61d2b53ec853fe6725fa

Reply:
Done - thanks!